hummingbot/core/data_type/order_book_tracker_data_source.py

```diff
@@ -87,7 +87,8 @@ async def listen_for_subscriptions(self):
                 )
                 await self._sleep(5.0)
             finally:
-                ws and await ws.disconnect()
+                # Replacing line with function to override for Bittrex
```

Comment:
I dont think this is exclusively for Bittrex. Nonetheless, I feel the function is useful for connectors that requires their own ws client.

Reply:
Updated the comment.